apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/PooledSession.cs

```diff
@@ -168,69 +168,63 @@ public void ReleaseToPool(bool forceDelete)
         /// </summary>
         /// <param name="request">The commit request. Must not be null. The request will be modified with session and transaction details
         /// from this object.</param>
-        /// <param name="timeoutSeconds">The timeout for this RPC, in seconds.</param>
-        /// <param name="cancellationToken">An optional token for canceling the call.</param>
+        /// <param name="callSettings">If not null, applies overrides to this RPC call.</param>
```

Comment:
Yes, that's right. Even if callSettings is non-null, it's merged with Client.Settings.XYZ.

This basically matches what the underlying SpannerClient methods have - but these parameters are required rather than being optional. We don't expect them to be called by many people, so it makes sense to make them pretty strict.